### PR TITLE
ref(ui):  Added tooltip - Datascrubbers v2 - Part 1

### PR DESCRIPTION
**Pull request by priscilawebdev**
**Type:** Refactor

**Description:**  

- converted `annotated.jsx` and `sdk.jsx` to typescript 
- renamed `sdk.tsx` to `eventSdk.tsx`
- added the Annotated Tooltip in the `eventSdk.tsx` component

You can read more about this refactor here: https://app.asana.com/0/1155078590577881/1158903109364929

**Result:**

image

### CODE REVIEW

File: src/sentry/static/sentry/app/components/events/eventSdk.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import EventDataSection from 'app/components/events/eventDataSection';
+import Annotated from 'app/components/events/meta/annotated';
+import {t} from 'app/locale';
+
+type Props = {
+  event: SentryTypes.Event;
+};
+
+const EventSdk = ({event: {sdk: data}}: Props) => (
+  <EventDataSection type="sdk" title={t('SDK')} wrapTitle>
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/events/eventSdk.tsx]: `wrapTitle` is defaulted to true.

**Review by markstory — APPROVED** [review]

File: src/sentry/static/sentry/app/components/events/eventSdk.tsx (hunk 1 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/eventSdk.tsx]: I'll remove it...thanks :)

Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/events/eventSdk.tsx]:
```suggestion
  <EventDataSection type="sdk" title={t('SDK')}>
```

**Review by priscilawebdev — COMMENTED** [review]
.

**Pull request merged**